src/components/InstancesList/InstancesList.js

```diff
@@ -372,15 +381,18 @@ class InstancesView extends React.Component {
     this.setState({ showErrorModal: false });
   };
 
-  toggleRowSelection = rowId => {
+  toggleRowSelection = ({
+    id: rowId,
+    ...rowData
```

Comment:
```suggestion
    ...rowData,
```

The same [here](https://github.com/folio-org/ui-inventory/pull/1256/files#diff-0399c7f70f7645c1cfd8ef8c3b7f167b29a8dc385e86b129f0f46ac896a02144R448) and [here](https://github.com/folio-org/ui-inventory/pull/1256/files#diff-0399c7f70f7645c1cfd8ef8c3b7f167b29a8dc385e86b129f0f46ac896a02144R456).